src/Command/ReferenceCommand.php

```diff
@@ -13,111 +13,95 @@
 
 namespace ApiPlatform\PDGBundle\Command;
 
+use ApiPlatform\PDGBundle\Parser\ClassParser;
 use ApiPlatform\PDGBundle\Services\ConfigurationHandler;
-use ApiPlatform\PDGBundle\Services\Reference\OutputFormatter;
-use ApiPlatform\PDGBundle\Services\Reference\PhpDocHelper;
-use ApiPlatform\PDGBundle\Services\Reference\Reflection\ReflectionHelper;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 use Symfony\Component\Config\Definition\Dumper\YamlReferenceDumper;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Twig\Environment;
 
 final class ReferenceCommand extends Command
 {
-    private \ReflectionClass $reflectionClass;
+    use CommandTrait;
 
     public function __construct(
-        private readonly PhpDocHelper $phpDocHelper,
-        private readonly ReflectionHelper $reflectionHelper,
-        private readonly OutputFormatter $outputFormatter,
-        private readonly ConfigurationHandler $configuration
+        private readonly ConfigurationHandler $configuration,
+        private readonly Environment $environment,
+        private readonly string $templatePath
     ) {
         parent::__construct(name: 'reference');
     }
 
     protected function configure(): void
     {
         $this
+            ->setDescription('Creates a reference documentation for a PHP class')
             ->addArgument(name: 'filename', mode: InputArgument::REQUIRED)
+            // todo remove output as it's already in the configuration file?
```

Comment:
the point was to be able to run these without any configuration but we probably went a bit further then I expected :)